tests/test_telemetry.py

```diff
@@ -614,6 +614,10 @@ def test_app_started_product_info(self):
                     "appsec" in products
                 ), "Product information is not accurately reported by telemetry on app-started event"
 
+    @irrelevant(
+        library="dotnet",
+        reason="dd-trace-dotnet already has this, will re-enable when this automatically updates the dd-go files.",
```

Comment:
If I'm not wrong,  the test in .NET is not exactly this one : the present test check that keys used are present in dd-go repo, while The test on dd-trace-dotnet repo does not fail if the key is missing on dd-go.

If i'm right, could you just keep the message `re-enable when this automatically updates the dd-go files.` ? I would not like to let someone think that this test could be also skipped for other repo for a bad reason.

Reply:
Like so?

```suggestion
        reason="Re-enable when this automatically updates the dd-go files.",
```

Reply:
But yes, the test in `dd-trace-dotnet` is a _copy_ of `dd-go`'s file unlike this one here

Reply:
> is a copy of `dd-go`'s file

Is it really? I tried copying the file recently and it has lots of diffs. I think we just add individual setting in the copy in `dd-trace-dotnet` without pull the copy from `dd-go`.

Reply:
A copy made at some other point in time in the past and rarely synced😄